.circleci/config.yml

```diff
@@ -55,6 +55,7 @@ jobs:
           fi
 
           docker run --name happa-branch -p 8000:8000 -d quay.io/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME:$CIRCLE_SHA1
+          sleep 2
```

Comment:
The new startup script is "slow" enough (blaming `jq` here I think) that the curl command now failes with error code 35 (SSL connection error). 

2 seconds is enough for the sed substitutions happen and for the nginx server to definitely be up.